Web-Scraping/Times_of_india/Times_of_india.py

```diff
@@ -0,0 +1,37 @@
+import requests
+import datetime
+from bs4 import BeautifulSoup
+
+url = "http://timesofindia.indiatimes.com/"
+
+# Use requests library to get html from artist's page
+response = requests.get(url)
+# Make the html soup object
+soup = BeautifulSoup(response.content, 'html.parser')
+
+print("\t!!!**The Times of India**!!!")
+today = datetime.date.today()
+print(today.strftime('\tThe date %d, %b %Y'))
+
+# scrping times of India in four domains:
+print("\n\t\t****Flash news****")
```

Comment:
edit this to `"\n\t\t**** Flash news ****"`

Reply:
Done @Neilblaze